examples/google-vertex-system-instruction/README.md

```
@@ -0,0 +1,55 @@
+2. Run the evaluation:
+
+```bash
+promptfoo eval
+```
+
+## System Instruction Format
+
+The system instruction can be provided in multiple ways:
+
+1. **Direct text in config:**
+   ```yaml
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)